lib/models/debugs_bunny/trace.rb

```diff
@@ -29,6 +29,12 @@ class Trace < DebugsBunny::ApplicationRecord
       created_after(time)
     }
 
+    after_rollback :rollback_callback
+
+    def rollback_callback
+      self.class.create({ **attributes.symbolize_keys, dump: dump })
```

Comment:
this could be dangerous if this create is the one that triggers the rollback

Reply:
You're right, there's always a risk that the `create` can throw an error, in which case we won't actually create a trace in this callback. But I think this is essentially the same risk we take with pretty much any record create. Do you recommend a different approach?

Reply:
My concern is that it will lead to an infinite loop of rollbacks callbacks

Reply:
I am ok with losing the trace in that case

Reply:
I'll do some local testing to see what happens when this callback throws an error

Reply:
My findings:

If `rollback_callback` throws an error, e.g. `self.class.create` throws an error, then the error is simply propagated through the call stack like any regular error. It doesn't call this callback again. This makes sense: when specifying a callback to `after_rollback`, the callback is only called at the point that the transaction is rolled back; it is only called once (per transaction). It is not called at the time an error occurs. See https://guides.rubyonrails.org/active_record_callbacks.html#transaction-callbacks for details.

However, if you had a contrived callback that itself defines a transaction:
```
self.class.transaction do
  self.class.create({ **attributes.symbolize_keys, dump: dump })
   raise StandardError
end
```
you get `SystemStackError: stack level too deep` as one would expect.